utilities/production_params.R

```diff
@@ -78,17 +78,36 @@ state_corrections_params <- zookeeper::default_state_params(
 state_corrector <- zookeeper::make_state_corrector(
   params = state_corrections_params,
   # data, locations, times to do special correction processing
-  manual_flags = tibble::tibble(
+  manual_flags = tibble(
```

Comment:
We actually need the `tibble::tibble()` version or this will fail. No packages are loaded with `library()`

Reply:
Oops, a bit too mindless with my copy-pasting.  Should be fixed now.